src/templates/partials/dovecot.hbs

```diff
@@ -5,8 +5,8 @@ ssl = required
 ssl_cert = </path/to/signed_cert_plus_intermediates
 ssl_key = </path/to/private_key
 {{#if output.usesDhe}}
-
-  {{#if (minver "2.3.0" form.serverVersion)}}
+  {{#if (minver "2.3.3" form.serverVersion)}}
+  {{else if (minver "2.3.0" form.serverVersion)}}
 # {{output.dhCommand}} > /path/to/dhparam
 ssl_dh = </path/to/dhparam
```

Comment:
@gene1wood Shouldn't this be left out only for `modern`? Since `intermediate` and `old` use DHE suites, they should still benefit from these dhparams(?)

Reply:
Actually I think this should be reverted. Since it's already behind a `usesDhe` test (that's `true` only when any `DHE-*` is included in the output suites), it won't be output for `modern` where it's not needed, and for `intermediate` and `old` the 2.2.x vs. 2.3.x ways of `ssl_dh*` was correct.

Reply:
What _might be needed_ is between v2.3–v2.3.3 adding the dhparams to the output always, unconditionally even when no DHE suites are present, as the docs state:

> "From version 2.3, you must specify path to DH parameters file using:"

(which is somewhat ambiguous, it's not clear from the docs that one must specify the path only if interested in own dhparams, or simply _always_…)

Reply:
@janbrasna Can you open an issue on this so we can track it outside this PR and determine what change is needed? Maybe just copy paste in what you wrote above?

Reply:
@gene1wood Worry not, will do once I have enough info on the last bit — I'm keeping it in mind.